mandatory/2-function-creation.js

```diff
@@ -1,11 +1,11 @@
 /*
 Write a function that:
-- takes an array of strings as input
-- removes any spaces in the beginning or end of the strings
-- removes any forward slashes (/) in the strings
+- takes a string as input
```

Comment:
This probably makes more sense as a task however as this in JS1-2 it makes sense that they should be working with Arrays in this task. 

Is it too much trouble to change it back to using arrays?

Reply:
Done - I'll update the next week's homework to also say:
```
NOTE: This exercise is the same as one you did last week - try to do it again using things you learnt this week.
Think about what is better about this solution than your one last week, and what is worse.
```
because I think it's worth repeating when they know `map`.